src/instruments/src/PFD/FMA.tsx

```diff
@@ -1,567 +1,1164 @@
+                this.c1Cell.instance.displayModeChangedPath();
+                this.BC1Cell.instance.displayModeChangedPath();
+            }
+        });
+    }
+
+    render(): VNode {
         return (
             <g>
-                <path className="NormalStroke White" d="m25.114 1.8143v13.506h-16.952v-13.506z" />
-                <text className="FontMedium MiddleAlign White" x="16.782249" y="7.1280665">MAN</text>
-                <text className="FontMedium MiddleAlign White" x="16.869141" y="14.351689">DTO</text>
+                <A1A2Cell bus={this.props.bus} />
+
+                <g ref={this.cellsToHide}>
+                    <B1Cell ref={this.b1Cell} bus={this.props.bus} />
+                    <C1Cell ref={this.c1Cell} bus={this.props.bus} />
+                    <D1D2Cell ref={this.D1D2Cell} bus={this.props.bus} />
+                    <BC1Cell ref={this.BC1Cell} bus={this.props.bus} />
+                </g>
+                <E1Cell bus={this.props.bus} />
             </g>
         );
-    } if (AThrMode === 5) {
+    }
+}
+
+class Row2 extends DisplayComponent<{bus:EventBus, isAttExcessive: Subscribable<boolean>}> {
+    private cellsToHide = FSComponent.createRef<SVGGElement>();
+
+    onAfterRender(node: VNode): void {
+        super.onAfterRender(node);
+
+        this.props.isAttExcessive.sub((a) => {
+            if (a) {
+                this.cellsToHide.instance.style.visibility = 'hidden';
+            } else {
+                this.cellsToHide.instance.style.visibility = 'visible';
+            }
+        });
+    }
+
+    render(): VNode {
         return (
             <g>
-                <path className="NormalStroke White" d="m25.114 1.8143v13.506h-16.952v-13.506z" />
-                <text className="FontMedium MiddleAlign White" x="16.782249" y="7.1280665">MAN</text>
-                <text className="FontMedium MiddleAlign White" x="16.869141" y="14.351689">MCT</text>
+                <A2Cell bus={this.props.bus} />
+                <g ref={this.cellsToHide}>
+                    <B2Cell bus={this.props.bus} />
+                    <C2Cell bus={this.props.bus} />
+                </g>
+                <E2Cell bus={this.props.bus} />
             </g>
         );
-    } if (AThrMode === 6) {
+    }
+}
+
+class A2Cell extends DisplayComponent<{ bus:EventBus }> {
+    private text = Subject.create('');
+
+    private className = Subject.create('FontMedium MiddleAlign Cyan');
+
+    private autoBrkRef = FSComponent.createRef<SVGTextElement>();
+
+    onAfterRender(node: VNode): void {
+        super.onAfterRender(node);
+
+        const sub = this.props.bus.getSubscriber<PFDSimvars>();
+
+        sub.on('autoBrakeMode').whenChanged().handle((am) => {
+            switch (am) {
+            case 0:
+                this.text.set('');
+                break;
+            case 1:
+                this.text.set('BRK LO');
+                break;
+            case 2:
+                this.text.set('BRK MED');
```

Comment:
There are spaces missing here at the end of the message, which are there to move the whole message to the left.
I think it was only one space at the end, but better look at the old message string to confirm